packages/wrangler/src/queues/cli/commands/consumer/worker/remove.ts

```diff
@@ -26,6 +26,6 @@ export async function handler(
 	const config = readConfig(args.config, args);
 
 	logger.log(`Removing consumer from queue ${args.queueName}.`);
-	await deleteConsumer(config, args.queueName, args.scriptName, args.env);
+	await deleteWorkerConsumer(config, args.queueName, args.scriptName);
```

Comment:
doesn't this still need the `env` to find the correct consumer id?

Reply:
Did some parkour on this, should correctly find the ID of service workers now.